### PR TITLE
Add docs for git and SuperTest

### PR DESCRIPTION
**Pull request by thaigillespie**

### CODE REVIEW

File: guides/testing/supertest.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Supertest
+
```
Comment (godwinpang) [inline guides/testing/supertest.md:2]: probably good to explain that we use supertest for e2e testing :)

**Review by godwinpang — CHANGES_REQUESTED** [review]

File: guides/testing/supertest.md (hunk 1 above)
Comment (thaigillespie) [reply guides/testing/supertest.md:2]: Ah yeah that's true, let me add that

Comment (thaigillespie) [reply guides/testing/supertest.md:2]: Added @godwinpang

**Comment by godwinpang** [comment]
@frankyyzy can you take a look at this PR and approve it if the docs look good?

**Review by godwinpang — APPROVED** [review]

**Pull request merged**